arelle/plugin/validate/HMRC/__init__.py

```diff
@@ -167,6 +172,7 @@ def validateXbrlStart(val, parameters=None, *args, **kwargs):
             if ns.startswith("http://www.xbrl.org/uk/char/"): val.txmyType = "charities"
             elif ns.startswith("http://www.xbrl.org/uk/gaap/"): val.txmyType = "ukGAAP"
             elif ns.startswith("http://www.xbrl.org/uk/ifrs/"): val.txmyType = "ukIFRS"
+            elif FRC_2022_NAMESPACE.search(ns): val.txmyType = "FRS-2022"
```

Comment:
Is it possible to look for defined concepts in the taxonomy instead?  If the set of required concepts doesn't change again for several years, it would be nice to not need to update the regex annually.